src/bootstrap/investment-sheet-boostrapper.js

```diff
@@ -6,46 +6,60 @@ class InvestmentSheetBootstrapper extends BaseSheetBootstrapper {
 
   createSheet() {
     const numRows = 200;
+    const letterColDate = colToLetter(TransactionsSheet.columnDate);
+    const letterColSource = colToLetter(TransactionsSheet.columnSource);
+    const letterColIncome = colToLetter(TransactionsSheet.columnIsIncome);
+    const letterColInvestment = colToLetter(TransactionsSheet.columnIsInvestment);
     this.investmentsTable = new DataTable(
       this.sheet,
       2,
       2,
       'Investment',
-      ['Date', 'Amount', 'Description', 'Category', 'Source'],
+      ['Date', 'Amount', 'Description', 'Category', 'Classificator', 'Source'],
       numRows
     )
       .withDataValidationRules([new ColumnValidationRule(5, this.categoryValidationRule)])
```

Comment:
This should be removed. There should be no category editing as part of this sheet.